### PR TITLE
fix(dockerimage): move from openjdk docker image to debian image and amazon-corretto

### PR DESCRIPTION
**Pull request by mariusgheorghies**
move from openjdk to amazon-corretto  ~debian default openjdk-17-jdk~ packages
see: https://github.com/docker-library/openjdk/issues/493#issuecomment-1105505718

upgrade base docker image from stretch to bullseye also

### CODE REVIEW

**Comment by mariusgheorghies** [comment]
> @mariusgheorghies how was this build tested ?

I built it locally and run it. No more than that

**Comment by bluestreak01** [comment]
thank you! This looks like a thorough build!

File: core/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+FROM debian:bullseye
 
 RUN apt-get update \
-  && apt-get install git curl -y
+  && apt-get install -y git curl wget software-properties-common unzip \
+  openjdk-17-jdk=17.0.3+7-1~deb11u1 openjdk-17-jre=17.0.3+7-1~deb11u1 \
+  openjdk-17-jdk-headless=17.0.3+7-1~deb11u1 openjdk-17-jre-headless=17.0.3+7-1~deb11u1
 
+
+ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
```
Comment (bluestreak01) [inline core/Dockerfile:9]: we also build ARM image, the build command can be found here:

https://github.com/questdb/questdb/tree/master/artifacts/release

as it stands `amd64` will not work for ARM

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: core/Dockerfile (hunk 1 above)
Comment (bluestreak01) [reply core/Dockerfile:9]: image

Comment (mariusgheorghies) [reply core/Dockerfile:9]: very good catch. fixing now

Comment (mariusgheorghies) [reply core/Dockerfile]: @bluestreak01 this should be fixed now. Can you try it now?

Comment (bluestreak01) [reply core/Dockerfile]: builds fine now, thank you! 🎉

**Review by bluestreak01 — APPROVED** [review]

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
docker image is massive:

image

**Comment by mariusgheorghies** [comment]
> docker image is massive:
ya i saw :( i guess the jvm comes with debug symbols. I will try to see if i can strip it or install jdk from another source

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**